src/rpc/request.rs

```diff
@@ -400,6 +400,9 @@ pub enum Request {
     // #[api(type = 105)]
     // #[display("list_offer_ids()")]
     // ListOfferIds,
+    #[api(type = 105)]
+    #[display("list_offers()")]
```

Comment:
`list_listens` so we get the correct `debug` level log for the request.

Reply:
thanks!